src/utils/shell/powerShellScripts.ts

```
@@ -0,0 +1,249 @@
+            Where-Object { $_ -like "$wordToComplete*" } |
+            ForEach-Object { New-GpxCompletionResult $_ }
+        return
+    }
+
+    if ($command -in $profileCommands) {
+        Get-GpxProfileNames |
+            Where-Object { $_ -like "$wordToComplete*" } |
+            ForEach-Object { New-GpxCompletionResult $_ }
+    }
+}
+    Register-ArgumentCompleter -Native -CommandName gpx -ScriptBlock $scriptblock`;
```

ESLint

[error] Unreachable code.

(no-unreachable)